### ISSUE TITLE
style prop error lacks context

### ISSUE THREAD

**Issue by dantman** [facebook/react#4168]
This error should probably have the "please check the render method of ???" or some other message to give context.
```
Uncaught Error: Invariant Violation: The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.
```

**Comment by spicyj** [comment]
I guess so, but I'm curious how you ended up in a situation where this wasn't obvious. In most cases I'd imagine that you would have just written the code seconds earlier so it would be pretty obvious?

**Comment by dantman** [comment]
Yes, that was the case. I just thought it would be good to have context like the warnings do.

Also it might not be obvious if you ported a bunch of templates all at once. Which actually would be reasonable in my case since what I'm doing is porting a bunch of stuff I wrote using Ractive over to React.

The only real reason I ended up running into this now rather than much earlier. Is simply because I avoid using the style attribute in my templates as much as possible. (this one was a simple `display: none;`)
